### DISCUSSION TITLE
Regarding Memory Limit

### DISCUSSION THREAD

**Discussion by xxccxcxc** [Xilinx/fpga24_routing_contest#36] [Q&A] [answered]
What's the memory limit of the machine that will be used for the contest evaluations?

**Comment by eddieh-xlnx** [comment, marked as answer]
Hi @xxccxcxc. The [Key Details](https://xilinx.github.io/fpga24_routing_contest/details.html#key-details) section of the website currently states that:
> Contestants can expect to be evaluated on an AMD multi-core Linux platform with >=32 cores, >=64GB RAM, and no internet connectivity.

Currently, we expect that 64GB RAM should be more than sufficient for the target device and are leaning to making this the upper limit.

**Comment by codepilot** [comment]
How much memory is in the GPU(s)?

Daniel

**Reply by eddieh-xlnx** [reply]
There's 64GB of memory on the AMD Instinct MI210 accelerator.
